data.json

```
@@ -1,4 +1,48 @@
 [
+  {
+    "command": "",
+    "description": "A package built to be a fast and flexible pure-Julia library for handling delimited text files.",
+    "kind": "header",
+    "package": "CSV",
+    "tags": [
+      "csv",
+      "delimited-files"
+    ]
+  },
+  {
+    "command": "CSV.Rows(source)",
+    "description": "Read a csv input and return a CSV.Rows object.",
+    "kind": "snippet",
```

```suggestion
    "command": "for row in CSV.Rows(source)\n      # do something\n  end",
    "description": "Read a csv input iterating over its rows. Only the current row values are buffered.",
    "kind": "snippet",
```
Since the command is a bit more complicated, I added more information.